src/main/java/org/tndata/android/compass/activity/ActionActivity.java

```diff
@@ -434,6 +440,62 @@ public void onClick(View v) {
     }
 
     /**
+     * Detect if the action belongs to a user and has a "datetime" external resource.
+     * See also: sendToCalendar.
+     *
+     * @return boolean
+     */
+    private boolean hasDatetimeExternalResource() {
+        if(isUserAction()) {
+            UserAction userAction = (UserAction) mAction;
+            String externalResource = userAction.getExternalResource();
+            String externalResourceName = userAction.getExternalResourceName();
```

Comment:
That would also deprecate this method: https://github.com/tndatacommons/android-app/blob/master/src/main/java/org/tndata/android/compass/util/CompassUtil.java#L147